setup/includes/upgrades/common/3.0.0-create-system-settings.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Common upgrade script for System Setting
+ *
+ * @var modX $modx
+ * @package setup
+ */
+
+use MODX\Revolution\modSystemSetting;
+
+$keys = ['fast_search_in_content','fast_search_result_max'];
```

Comment:
```suggestion
$keys = [
    'fast_search_in_content',
    'fast_search_result_max'
];
```

Because options are pretty long, easier to read them when they in the column.